packages/sdk/tasks/deposit.ts

```diff
@@ -108,6 +119,60 @@ task('deposit', 'Deposits WETH9 onto L2.')
       bedrock: true,
     })
 
+    const opBalanceBefore = await signer.provider.getBalance(
+      OptimismPortal.address
+    )
+
+    // Deposit ETH
+    console.log('Depositing ETH through StandardBridge')
+    const ethDeposit = await messenger.depositETH(utils.parseEther('1'))
+    const messageReceipt = await ethDeposit.wait()
+    if (messageReceipt.status !== 1) {
+      throw new Error('deposit failed')
+    }
+    console.log('Deposit complete')
+
+    const opBalanceAfter = await signer.provider.getBalance(
+      OptimismPortal.address
+    )
+
+    if (!opBalanceBefore.add(utils.parseEther('1')).eq(opBalanceAfter)) {
```

Comment:
This checks the portal balance, but not the actual balance on L2. Intentional?

Reply:
This is intentional, the call in the `OptimismPortal` sometimes doesn't actually transfer the ETH